python/ray/data/dataset.py

```diff
@@ -2130,16 +2131,17 @@ def unique(self, column: str) -> List[Any]:
             >>> train_ds.sort("sepal length (cm)").take(1)  # Sort to make it deterministic
             [{'sepal length (cm)': 4.3, ..., 'variety': 'Setosa'}]
 
-        Time complexity: O(dataset size * log(dataset size / parallelism))
+        Time complexity: O(dataset size / parallelism)
 
         Args:
             column: The column to collect unique elements over.
+            ignore_nulls: Whether to ignore nulls when computing the unique elements.
```

Comment:
Can you elaborate on the motivation of adding this argument? 
It doesn't seem that useful.
E.g. None is not special compared with other values. Without this arg, users can also easily remove None from the returned values.

Reply:
>motivation 

I was following the function args of other aggregations. It's actually needed in `_build_multicolumn_aggs()` (which could do some clean up re *args..) (maybe a separate PR?)

Reply:
Thanks for explaining. 
I think for `sum/min/max`, `ignore_nulls` makes sense because null values are invalid in these operations. 
For `unique`, I prefer not adding ignore_nulls for now unless there is a strong reason. 
The kwargs cleanup looks good.